index.html

```diff
@@ -39,94 +43,111 @@ <h1 class="greet">Hey There. <br> I’m Alex</h1>
   </section>
```

Comment:
- [ ] The `background-image` of the headline section is much bigger in Figma than the one in your design, kindly match the template 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213469718-7486a5c1-81fd-440d-868c-178702e4afb5.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/213469750-18f8f1db-19e0-48c7-9aab-ca8e8dbbd33d.png)